lib/hammer_cli/settings.rb

```diff
@@ -82,6 +82,14 @@ def self.deep_merge!(h, other_h)
     def self.symbolize_hash(h)
       h = h.inject({}) { |sym_hash,(k,v)| sym_hash.update(k.to_sym => v) }
     end
+    
+    def self.get_defaults_by_key(key, option)
```

Comment:
I'd prefer to keep settings untouched. Could this be moved to separate class and use Settings just as a storage?